### PR TITLE
Setup CLI commands for the Spinup Framework

### PR DESCRIPTION
**Pull request by Teknologica**
This PR setups the first iteration of the CLI tool and will eventually publish `rebilly-spinup` under `@rebilly/spinup-cli`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
-# TODO
+#Rebilly Spinup
+
+Monorepository encompasing all packages relevant to the Spinup Framework.
```
Comment (mathewsonke) [inline README.md]:
```suggestion
Monorepository encompassing all packages relevant to the Spinup Framework.
```

File: packages/spinup-cli/package.json [hunk 2]
```diff
@@ -1,19 +1,14 @@
 {
-  "version": "0.0.1",
-  "name": "@gridsome/spinup-cli",
-  "description": "A command line tool for creating portals at Rebilly.",
-  "homepage": "https://github.com/Rebilly/rebilly-spinup/tree/master/packages/spinup-cli#readme",
-  "repository": "https://github.com/Rebilly/rebilly-spinup/tree/master/packages/spinup-cli",
+  "name": "@rebilly/spinup-cli",
+  "version": "0.1.2",
```
Comment (mathewsonke) [inline packages/spinup-cli/package.json:3]: why the jump to `0.1.2`, shouldn't it be `0.1.0`?

**Review by mathewsonke — CHANGES_REQUESTED** [review]
Looks excellent, just two veeery minor things

File: packages/spinup-cli/package.json (hunk 2 above)
Comment (Teknologica) [reply packages/spinup-cli/package.json:3]: I was testing to make sure the global install worked and picked up the version number between updates.

**Review by mathewsonke — APPROVED** [review]

**Pull request merged**